observation_portal/requestgroups/serializers.py

```diff
@@ -299,6 +299,9 @@ def validate(self, data):
         if len(data['instrument_configs']) > 1 and data['type'] in ['SCRIPT', 'SKY_FLAT']:
             raise serializers.ValidationError(_(f'Multiple instrument configs are not allowed for type {data["type"]}'))
 
+        if len(data['instrument_configs']) > 1 and instrument_type == '2M0-SCICAM-MUSCAT':
+            raise serializers.ValidationError(_('Multiple instrument configs are not allowed for muscat'))
```

Comment:
I would change this to "the Muscat instrument" or "the instrument 2M0-SCICAM-MUSCAT" to be a little more explicit